exonum/src/storage/options.rs

```diff
@@ -14,16 +14,25 @@
 
 //! Abstract settings for databases.
 
-/// Options for database.
+/// Options for the database.
+///
+/// These parameters apply to the underlying database of Exonum, currently RocksDB.
 #[derive(Debug, Clone, Copy, Serialize, Deserialize)]
 pub struct DbOptions {
-    /// Number of open files that can be used by the DB.
+    /// Number of open files that can be used by the database. The underlying
+    /// database opens multiple files during operation. If your system has a
+    /// limit on the number of files which can be open simultaneously, you can
+    /// adjust this option to match the limit. Note, that limiting the number
+    /// of simultaneously open files might slow down the speed of database operation.
     ///
-    /// Defaults to `None`, which means opened files are always kept open.
+    /// Defaults to `None`, meaning that the number of open files is unlimited.
     pub max_open_files: Option<i32>,
-    /// Whether create database or not, if it's missing.
+    /// Whether the system should create a database or not, if it's missing.
     ///
-    /// Defaults to `true`.
+    /// Defaults to `true`. This option applies to the cases when a node was
```

Comment:
I'd move "Defaults to `true`" to a separate paragraph at the end of the doc comment.